index.css

```diff
@@ -1,3 +1,4 @@
 div {
   background-color: red;
+  color: beige;
```

Comment:
hey! actually I don't want the color to be beige.. I want it to be blue.

Reply:
I give in